cgcreate: add page (#6768)
tldr-pages/tldr: pages/linux/cgcreate.md

# cgcreate

> Create cgroups, used to limite, mesure, and control resources used by processes.

> > cgroups types can be `memory`, `cpu`, `net_cls`, etc.

Reviewer: Where should I open an issue about this? `tldr-lint`, `tldr-bot` or here?

> > cgroups types can be `memory`, `cpu`, `net_cls`, etc.

Reviewer 2: tldr-lint @marchersimon I think

# cgcreate

> Create cgroups, used to limite, mesure, and control resources used by processes.
> You can create cgroups with types like `memory`, `cpu`, `net_cls`, etc.
> More information: <https://www.kernel.org/doc/Documentation/cgroup-v1/>.

> 
> - To install use:
> 
> `apt install cgroup-lite cgroup-tools`
> 

Reviewer: This completely depends on the Distro being used.
```suggestion
```

> - To create a new group use:

Reviewer: ```suggestion
- Create a new group:
```

> `cgcreate -g {{groupType}}:{{groupName}}`

Reviewer: ```suggestion
`cgcreate -g {{group_type}}:{{group_name}}`
```

> - To create a new group with multiple croup types:

Reviewer: ```suggestion
- Create a new group with multiple cgroup types:
```

> `cgcreate -g {{groupType1}},{{groupType2}}:{{groupName}}`

Reviewer: ```suggestion
`cgcreate -g {{group_type1}},{{group_type2}}:{{group_name}}`
```

> - To create subgroups:

Reviewer: ```suggestion
- Create a subgroup:
```

> `mkdir /sys/fs/cgroup/{{groupType}}/{{groupName}}/{{subgroupName}}`

Reviewer: ```suggestion
`mkdir /sys/fs/cgroup/{{group_type}}/{{group_name}}/{{subgroup_name}}`
```
